pom.xml

```diff
@@ -281,11 +281,11 @@
           <fail>true</fail>
           <failFast>false</failFast>
           <rules>
-            <dependencyConvergence></dependencyConvergence>
+            <dependencyConvergence />
```

Comment:
why?

Reply:
Cleaner, throwing warnings, kind of a code smell. Should I move that into a diff PR.

Reply:
I disagree that it's cleaner or a code smell. What is giving warnings on this?

Reply:
It's telling me it's an empty code blocks. In xml you use <x /> to represent an action rather than a block which it seems like this is. Using <x> </x> kind of says it has stuff inside of it. From what I know anyways, I might be wrong and I can change it back.

Reply:
Talking about actions and code blocks in xml doesn't make sense to me. XML is just a data format.

An XML parser shouldn't treat `<x></x>` and `<x/>` any differently

Reply:
See https://stackoverflow.com/questions/7231902/self-closing-tags-in-xml-files
It shouldn't treat it differently but from what I've seen it's `more correct` to use the self-closing tag in this instance.

Reply:
We don't really comment it too much. Trying to find more accurate convention for xml self closing tags.

Reply:
https://blogs.msdn.microsoft.com/ericwhite/2009/07/07/empty-elements-and-self-closing-tags/
Can also be seen here, seems like the popular opinion is that it provides a cleaner workspace. I can change it back if you'd like.

Reply:
I'm not sure how you get that from that post:
> Further, per the specification, “the empty-element tag SHOULD be used, and SHOULD only be used, for elements which are declared EMPTY”.  This means that it’s always safe to serialize an empty element as a self-closing element, but sometimes it’s not correct to serialize an empty element with a start and end tag.

and here's the style used in the pom for maven itself: https://github.com/apache/maven/blob/master/pom.xml#L529

Reply:
sorry, I misread this whole change wrong from the start. I thought it was changing from `<x/>` to `<x></x>`.

`<x/>` is good